src/main/java/org/frc5687/rapidreact/OI.java

```diff
@@ -67,17 +43,18 @@ public double getDriveY() {
     }
 
     public double getDriveX() {
-        //        xIn = -getSpeedFromAxis(_leftJoystick, _leftJoystick.getXChannel());
-        xIn = -getSpeedFromAxis(_driverGamepad, Gamepad.Axes.LEFT_X.getNumber());
+        //Comment for gamepad control
+        xIn = -getSpeedFromAxis(_rotation, _rotation.getXChannel());
+        //Uncomment for gamepad control
+        //xIn = -getSpeedFromAxis(Gamepad, Gamepad.getXChannel());
         xIn = applyDeadband(xIn, Constants.DriveTrain.DEADBAND);
-
         double xOut = xIn / (Math.sqrt(yIn * yIn + (xIn * xIn)) + Constants.EPSILON);
         xOut = (xOut + (xIn * 2)) / 3.0;
         return xOut;
     }
 
     public double getRotationX() {
-        double speed = getSpeedFromAxis(_rightJoystick, _rightJoystick.getZChannel());
+        double speed = getSpeedFromAxis(_translation, _translation.getXChannel());
         speed = applyDeadband(speed, 0.2);
```

Comment:
`speed = applyDeadband(speed, 0.2);` Magic number, could be placed in constant. If it has a purpose like tolerance, no reason to not label it.  For `xOut = (xOut + (xIn * 2)) / 3.0;` the 3.0 was empirically tested and has no "name" we could fix this but I'm not sure what we would call it.